src/templates/ObjectMethodGetOriginalFieldValues.php

```diff
@@ -16,4 +16,23 @@ public function getOriginalFieldValues($mapped = true)
     }
 
     return $returnValues;
+}
+
+/**
+ * Return true if object was new before saving
+ */
+
+public function getWasNew()
+{
+    return $this->wasNew;
+}
+
+
+/**
+ * Return true if object was modified
+ */
+
+public function getWasModified()
```

Comment:
Change method name of "getWasModified" to "wasModified" to make it consistent with "isModified".